HACKING.md

```diff
@@ -360,15 +383,22 @@ This will create:
 
 1. the Jenkins controller imagestream,
 2. the Jenkins agent imagestream,
-3. the coreos-assembler imagestream, and
-4. the pipeline-config configmap.
+3. the Jenkins agent BuildConfig (if a root CA cert was provided),
+4. the coreos-assembler imagestream, and
+5. the pipeline-config configmap.
 
 We can now start a build of the Jenkins controller:
 
 ```
 oc start-build --follow jenkins
 ```
 
+If a root CA cert was provided, also start a build of the Jenkins agent:
+
+```
+oc start-build --follow jenkins-agent
+```
+
```

Comment:
Possibly. I do like that we only pay for it if it's needed, but I see the appeal of keeping it consistent. Though in the no-cert case, there's nothing to actually build. We could make it be a no-op build I guess (the Dockerfile templating there would get hacky, or maybe just have two separate manifest files), though it might be more complex than it's worth.